packages/subgraph/tests/registry/daoRegistry.test.ts

```diff
@@ -24,50 +25,55 @@ describe('DAORegistry', () => {
     // handle event
     handleDAORegistered(newDaoEvent);
 
-    let entityID = Address.fromString(DAO_ADDRESS).toHexString();
+    const daoAddress = Address.fromString(DAO_ADDRESS);
+    const daoEntityId = generateDaoEntityId(daoAddress);
 
     // checks
-    assert.fieldEquals('Dao', entityID, 'id', entityID);
+    assert.fieldEquals('Dao', daoEntityId, 'id', daoEntityId);
     assert.fieldEquals(
       'Dao',
-      entityID,
+      daoEntityId,
       'creator',
       Address.fromString(ADDRESS_ONE).toHexString()
     );
-    assert.fieldEquals('Dao', entityID, 'subdomain', 'mock-Dao');
+    assert.fieldEquals('Dao', daoEntityId, 'subdomain', 'mock-Dao');
     assert.fieldEquals(
       'Dao',
-      entityID,
+      daoEntityId,
       'createdAt',
       newDaoEvent.block.timestamp.toString()
     );
   });
 
   test("Don't store subdomain for blocklisted DAO", () => {
     // Using an already blocklisted address for mainnet. This is unlikely to change
-    let entityID = '0x16070493aa513f91fc8957f14b7b7c6c0c41fbac';
+    let blacklistedEntityId = '0x16070493aa513f91fc8957f14b7b7c6c0c41fbac';
```

Comment:
Use denylist/allowlist instead of blacklist/whitelist. See https://aragonassociation.atlassian.net/browse/APP-47 for the reasoning.

Reply:
fixed in [7e98a90](https://github.com/aragon/osx/pull/520/commits/7e98a90c608bf5e6dfaf7a66b02d887853de5407)